apps/pipelines/nodes/base.py

```diff
@@ -122,12 +124,23 @@ def clone(cls, state):
         return PipelineState(copied)
 
     @classmethod
-    def from_node_output(cls, node_name: str, node_id: str, output: Any = None, **kwargs) -> Self:
+    def from_node_output(
+        cls,
+        node_name: str,
+        node_id: str,
+        output: Any = None,
+        voice_provider_id: str | None = None,
+        synthetic_voice_id: str | None = None,
+        **kwargs,
+    ) -> Self:
         kwargs["outputs"] = {node_name: {"message": output, "node_id": node_id}}
         kwargs.setdefault("temp_state", {}).update({"outputs": {node_name: output}})
         if output is not None:
             kwargs["messages"] = [output]
-
+        if voice_provider_id is not None:
+            kwargs["voice_provider_id"] = voice_provider_id
+        if synthetic_voice_id is not None:
+            kwargs["synthetic_voice_id"] = synthetic_voice_id
```

Comment:
These changes don't seem necessary since we're already accepting arbitrary kwargs